sqlglot/generator.py

```diff
@@ -2850,12 +2850,18 @@ def respectnulls_sql(self, expression: exp.RespectNulls) -> str:
         return self._embed_ignore_nulls(expression, "RESPECT NULLS")
 
     def _embed_ignore_nulls(self, expression: exp.IgnoreNulls | exp.RespectNulls, text: str) -> str:
-        if self.IGNORE_NULLS_IN_FUNC:
-            this = expression.find(exp.AggFunc)
-            if this:
-                sql = self.sql(this)
-                sql = sql[:-1] + f" {text})"
-                return sql
+        if self.IGNORE_NULLS_IN_FUNC and not expression.args.get("inline"):
+            for klass in (exp.Order, exp.Limit):
```

Comment:
Interesting, so order will be closer to the arg than limit and that's why you look for it first in this loop with `find`?

Reply:
can we do this without introducing the `inline` argument? i think `meta` can work too right? we can avoid introducing the arg only for this

Reply:
yea, also realzide's ther's a having min by i'll do another pr for that

Reply:
I was looking at the having min / max earlier, i'm not sure if it's applicable to all agg funcs in BQ. I'd double check if it's only an `ANY_VALUE` thing because it may not be necessary. I'm saying this because we also don't support that syntax in general, just for `ANY_VALUE`, and that's also the only func they have in their examples where that clause is used.